content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -85,11 +101,12 @@ An opportunity's probability of success can be changed manually on the opportuni
 the probability number to edit it.
 
 .. important::
-   Manually changing the probability removes the automatic probability updates for that
-   opportunity. The probability will no longer update automatically as the opportunity moves
-   through each stage of the pipeline.
+   Manually changing the probability removes the automatic probability updates for that opportunity.
+   The probability will no longer update automatically as the opportunity moves through each stage
+   of the pipeline.
 
-To reactivate automatic probability, click on the gear icon next to the probability percentage.
+To reactivate automatic probability, click on the :icon:`fa-cog` Actions menu icon next to the
+probability percentage.
```

Comment:
```suggestion
To reactivate automatic probability, click on the :icon:`fa-cog` :guilabel:`(gear) icon next to the
:guilabel:`Probability` percentage.
```
Since this *technically* isn't the 'Actions' gear menu (that's located above the form, beside the 'Discard' and 'Save automatically' icons), I think it might be best to format it this way. And, since the term 'Probability' appears on the screen, I believe that should be given a GUI, as well.